### ISSUE TITLE
why it consume so many time in cycle 1 epoch 0

### ISSUE THREAD

**Issue by waibi8bo** [agaldran/lwnet#5]
**such as**
Cycle 1/40
  0%|                                                          | 0/50 [13:15<?, ?it/
100%|████████████████████████████████████████████████████| 50/50 [13:46<00:00, 16.52s/it, tr_loss_lr=nan/0.000002]
-------------------------  End of cycle, evaluating -------------------------
Train/Val Loss: nan/nan  -- Train/Val AUC: 0.4871/0.5019  -- Train/Val DICE: 0.0000/0.0000 -- LR=0.01

**but after that it seems to be normal（but i have no idea why loss and dice is abnormal and auc is so low）**
Cycle 2/40
100%|██████████████████████████████████████████████████████████████| 50/50 [00:30<00:00,  1.62it/s, tr_loss_lr=nan/0.000002]
-------------------------  End of cycle, evaluating -------------------------
Train/Val Loss: nan/nan  -- Train/Val AUC: 0.5153/0.5019  -- Train/Val DICE: 0.0000/0.0000 -- LR=0.01
Cycle 24/40
100%|██████████████████████████████████████████████████████████████| 50/50 [00:30<00:00,  1.62it/s, tr_loss_lr=nan/0.000002]
-------------------------  End of cycle, evaluating -------------------------
Train/Val Loss: nan/nan  -- Train/Val AUC: 0.5067/0.5019  -- Train/Val DICE: 0.0000/0.0000 -- LR=0.01
Cycle 40/40
100%|██████████████████████████████████████████████████████████████| 50/50 [00:30<00:00,  1.62it/s, tr_loss_lr=nan/0.000002]
-------------------------  End of cycle, evaluating -------------------------
Train/Val Loss: nan/nan  -- Train/Val AUC: 0.5225/0.5019  -- Train/Val DICE: 0.0000/0.0000 -- LR=0.01

**Comment by waibi8bo** [comment]
thank you!!!

**Comment by agaldran** [comment]
Hi,

That looks very weird. What is the exact command you used to start the training? I need to be able to reproduce it in order to understand what's going on.

Adrian

**Comment by waibi8bo** [comment]
thank you for your reply.
I solve that problem by creating a new environment instead of following the environment.txt .

**Comment by agaldran** [comment]
Ok great; have into account that if you use the requirements.txt in the repo you will see some torchvision warnings regarding interpolation on image transforms. Just upgrade to a newer torchvision to fix that, please.
